openstudiocore/src/energyplus/Test/ScheduleInterval_GTest.cpp

```diff
@@ -144,9 +147,112 @@ TEST_F(EnergyPlusFixture,ForwardTranslator_ScheduleFixedInterval_Hourly)
   EXPECT_EQ(8760, numUntils);
 }
 
+TEST_F(EnergyPlusFixture, ForwardTranslator_ScheduleFixedInterval_Hourly_Shifted)
+{
+  Vector values = linspace(1, 8760, 8760);
+
+  TimeSeries timeseries(DateTime(Date(MonthOfYear::Jan, 1)), Time(0, 1, 0), values, "");
+
+  Model model;
+
+  boost::optional<ScheduleInterval> scheduleInterval = ScheduleInterval::fromTimeSeries(timeseries, model);
+  ASSERT_TRUE(scheduleInterval);
+  EXPECT_TRUE(scheduleInterval->optionalCast<ScheduleFixedInterval>());
+
+  TimeSeries ts = scheduleInterval->timeSeries();
+  EXPECT_EQ(DateTime(Date(MonthOfYear::Jan, 1)), ts.firstReportDateTime());

```

Comment:
@macumber No, the time series was created with a first report date time of 1/1 00:00 and a 1 hour interval, so the time series starts at 12/31 23:00. The constructor that is getting used is this one:

https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/utilities/data/TimeSeries.hpp#L161

Since the time series isn't translated properly into the schedule object, it can't return the right time series. I'm not sure what the right time series is, but I don't think it is this one.